pkg/services/featuremgmt/registry.go

```diff
@@ -1635,6 +1635,13 @@ var (
 			Owner:        grafanaObservabilityTracesAndProfilingSquad,
 			FrontendOnly: true,
 		},
+		{
+			Name:        "reportingUseRawTimeRange",
+			Description: "Uses the original report or dashboard time range instead of making an absolute transformation",
+			Stage:       FeatureStageExperimental,
+			Owner:       grafanaSharingSquad,
+			Expression:  "false", // disabled by default
```

Comment:
```suggestion
			Expression:  "true", // disabled by default
```
I would enable it by default as it should fix the bug with images and mostly impact users with the `newPDFRendering`  feature flag enabled.

Reply:
finally, it is disabled by default because we would need the FF to be GA and we want it to be Public Preview. Rollout Plan will be created soon